dummyisolator/isolator.cpp

```diff
@@ -14,15 +14,17 @@
 #include <stout/hashmap.hpp>
 #include <stout/option.hpp>
 
-using namespace mesos;
+#include <list>
+#include <string>
+
+// TODO(nnielsen): Break into explicit using-declarations instead.
+using namespace mesos;  // NOLINT(build/namespaces)
 
 using mesos::slave::Isolator;
 
-class DummyIsolatorProcess : public mesos::slave::IsolatorProcess
-{
-public:
-  static Try<mesos::slave::Isolator*> create(const Parameters& parameters)
-  {
+class DummyIsolatorProcess : public mesos::slave::IsolatorProcess {
+ public:
+  static Try<mesos::slave::Isolator*> create(const Parameters& parameters) {
```

Comment:
That is an exception from the Google style guide (which the Mesos style guide should be describing). I don't have a strong preference, other than using something we can automate. cpplint (from Google) enforces the inline braces.